hack/test-stirunimage.sh

```diff
@@ -8,14 +8,14 @@ export PATH="$PWD/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH):$PAT
 
 function time_now()
 {
-  date +%s000
+    date +%s000
```

Comment:
I think it would be preferable to revert the whitespace changes and pursue them in a separate PR.